src/streamlit_app.py

```diff
@@ -49,8 +55,7 @@ async def main() -> None:
         st.rerun()
 
     if "agent_client" not in st.session_state:
-        agent_url = os.getenv("AGENT_URL", "http://localhost")
-        st.session_state.agent_client = AgentClient(agent_url)
+        st.session_state.agent_client = AgentClient()
```

Comment:
I think the `os.getenv` check is still needed, no? Maybe a missed revert when removing settings from the client side

Reply:
fixed